### PR TITLE
ci: add auto assigner to unassigned PRs

### PR DESCRIPTION
**Pull request by 0xbeny**
Regarding this issue #796

### CODE REVIEW

**Comment by moul** [comment]
The file is empty. Can you retry?

**Comment by 0xbeny** [comment]
@moul I've updated the file, also i'm going to test with new PRs

**Comment by moul** [comment]
We need to merge so that new PRs can trigger the new workflow.

You can test on a fork.

File: .github/workflows/auto-author-assign.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: Auto Author Assign
```
Comment (thehowl) [inline .github/workflows/auto-author-assign.yml:1]: Following the pattern of our existing workflows,

```suggestion
name: auto-author-assign
```

**Review by thehowl — COMMENTED** [review]
Looks good. This is basically identical as the example in the [action's readme](https://github.com/toshimaru/auto-author-assign), so after this I think it's safe to mark this PR as ready and then merge it.

**Review by moul — APPROVED** [review]

**Pull request merged**